recipe/gen_patch_json.py

```diff
@@ -691,7 +691,7 @@ def _gen_new_index(repodata, subdir):
         with suppress(ValueError):
             i = deps.index("cudatoolkit 11.2|11.2.*")
         if i >= 0:
-            deps[i] = "cudatoolkit >=11.2,<12"
+            deps[i] = "cudatoolkit >=11.2,<12.0a0"
```

Comment:
May have committed this change to quickly. Happy to keep or revert this

Reply:
That's fine. I was going to suggest using `pin_looser`, but that doesn't seem to be doing the correct thing.

Reply:
Ah ok. Yeah I tried that originally, but likely ran into the same issue. Guessing the format of the original version constraint was throwing it for some reason.